content/de/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,271 @@
+
+    ```shell
+    minikube addons list
+    ```
+
+    Ausgabe:
+
+    ```shell
+    addon-manager: enabled
+    coredns: disabled
+    dashboard: enabled
+    default-storageclass: enabled
+    efk: disabled
+    freshpod: disabled
+    heapster: disabled
+    ingress: disabled
+    kube-dns: enabled
+    metrics-server: disabled
+    nvidia-driver-installer: disabled
+    nvidia-gpu-device-plugin: disabled
+    registry: disabled
+    registry-creds: disabled
+    storage-provisioner: enabled
+    ```
+   
+2. Aktivieren Sie ein Addon, zum Beispiel `heapster`:
+
+    ```shell
+    minikube addons enable heapster
+    ```
+  
+    Ausgabe:
+
+    ```shell
+    heapster was successfully enabled
+    ```
+
+3. Sehen Sie sich den Pod und den Service an, den Sie gerade erstellt haben:
+
+    ```shell
+    kubectl get pod,svc -n kube-system
+    ```
+
+    Output:
+
+    ```shell
+    NAME                                        READY     STATUS    RESTARTS   AGE
+    pod/heapster-9jttx                          1/1       Running   0          26s
+    pod/influxdb-grafana-b29w8                  2/2       Running   0          26s
+    pod/kube-addon-manager-minikube             1/1       Running   0          34m
+    pod/kube-dns-6dcb57bcc8-gv7mw               3/3       Running   0          34m
+    pod/kubernetes-dashboard-5498ccf677-cgspw   1/1       Running   0          34m
+    pod/storage-provisioner                     1/1       Running   0          34m
+
+    NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+    service/heapster               ClusterIP   10.96.241.45    <none>        80/TCP              26s
+    service/kube-dns               ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP       34m
+    service/kubernetes-dashboard   NodePort    10.109.29.1     <none>        80:30000/TCP        34m
+    service/monitoring-grafana     NodePort    10.99.24.54     <none>        80:30002/TCP        26s
+    service/monitoring-influxdb    ClusterIP   10.111.169.94   <none>        8083/TCP,8086/TCP   26s
+    ```
+
+4. Deaktivieren Sie `heapster`:
+
+    ```shell
+    minikube addons disable heapster
+    ```
+  
+    Ausgabe:
+
+    ```shell
+    heapster was successfully disabled
+    ```
+
+## Aufräumen
+
+Jetzt können Sie die in Ihrem Cluster erstellten Ressourcen bereinigen:
+
+```shell
+kubectl delete service hello-node
+kubectl delete deployment hello-node
+```
+
+Stoppen Sie optional die virtuelle Minikube-Maschine (VM):
+
+```shell
+minikube stop
+```
+
+Löschen Sie optional die Minikube-VM:
+
+```shell
+minikube delete
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Lernen Sie mehr über [Deployment objects](/docs/concepts/workloads/controllers/deployment/).
+* Lernen Sie mehr über [Deploying applications](/docs/user-guide/deploying-applications/).
+* Lernen Sie mehr über [Service objects](/docs/concepts/services-networking/service/).
```

Comment:
Is it possible to translate `Deployment objects`, `Deploying applications` and `Service objects` as well to German? If it won't become clear, just leave it like this.